tf2onnx/tfonnx.py

```diff
@@ -1154,6 +1160,29 @@ def minmax_op(ctx, node, name, args):
     # handle this by doing something like:
     # y = min(x1, add(x2, sub(x1, x1))), where x1, x2 are the inputs and x2 is a scalar
     # this will create a tensor of zeros of the shape of x1, adds x2 to it (which broadcasts) and use that for min.
+    # support more dtype
+    supported_dtypes = [
+        onnx_pb.TensorProto.FLOAT,
+        onnx_pb.TensorProto.FLOAT16,
+        onnx_pb.TensorProto.DOUBLE
+    ]
+    target_dtype = onnx_pb.TensorProto.FLOAT
+    for inp in node.input:
+        dtype = ctx.get_dtype(inp)
+        if not dtype:
+            log.warning("dtype of %s is None", inp)
```

Comment:
print a warning won't help...
dtype SHOULD be here, unless we screwed up somewhere earlier...
how about make_sure?

Reply:
If input dtype is None while output dtype is given, this conversion can work.
Anyway, make_sure can throw the error early so that we can debug it more easily.